module2/homework/transformContainers/transform.cpp

```diff
@@ -0,0 +1,18 @@
+#include "transform.hpp"
+
+#include <algorithm>
```

Comment:
You should add here deque,list,map and string according to "Include-what-you-use" rule.